jm-first-project/src/jm_first_project/java_test.py

```
@@ -0,0 +1,10 @@
+import os
+import subprocess
+
+print("JAVA_HOME =", os.environ.get("JAVA_HOME"))
+
+try:
+    output = subprocess.check_output("java -version", stderr=subprocess.STDOUT, shell=True)
+    print(output.decode())
+except subprocess.CalledProcessError as e:
+    print("Java test failed:", e.output.decode())
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)